app/konnect/account-management/index.md

```diff
@@ -36,6 +42,7 @@ This page provides an overview of your current plan, including:
 * An estimated cost for the month (not available for trial accounts)
 * The total count of all services, services published to Dev Portal, and API calls for
 the current month
+* The number of regions (contact a sales representative to add regions)
```

Comment:
```suggestion
* The number of regions
```
You can ignore this suggestion, but this reads a bit too sales-heavy. Maybe we can rewrite line 53 to cover this case.